diff --git a/dist/index.cjs b/dist/index.cjs
index 635f1c0134274c89e7efbcae2adf3d9ecba575ee..0054b779ff7b550ecf6d4008e177a888003f9c92 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -977,7 +977,6 @@ async function discoverMetadataWithFallback(serverUrl, wellKnownType, fetchFn, o
 	else {
 		const wellKnownPath = buildWellKnownPath(wellKnownType, issuer.pathname);
 		url = new URL(wellKnownPath, opts?.metadataServerUrl ?? issuer);
-		url.search = issuer.search;
 	}
 	let response = await tryMetadataDiscovery(url, protocolVersion, fetchFn);
 	if (!opts?.metadataUrl && shouldAttemptFallback(response, issuer.pathname)) response = await tryMetadataDiscovery(new URL(`/.well-known/${wellKnownType}`, issuer), protocolVersion, fetchFn);
diff --git a/dist/index.mjs b/dist/index.mjs
index f02ce3ca394e826fc27c9848dbe9a214bf6ba7a9..8e27c9320e14ba8816fda752f250f6ddb1cc7ed0 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -974,7 +974,6 @@ async function discoverMetadataWithFallback(serverUrl, wellKnownType, fetchFn, o
 	else {
 		const wellKnownPath = buildWellKnownPath(wellKnownType, issuer.pathname);
 		url = new URL(wellKnownPath, opts?.metadataServerUrl ?? issuer);
-		url.search = issuer.search;
 	}
 	let response = await tryMetadataDiscovery(url, protocolVersion, fetchFn);
 	if (!opts?.metadataUrl && shouldAttemptFallback(response, issuer.pathname)) response = await tryMetadataDiscovery(new URL(`/.well-known/${wellKnownType}`, issuer), protocolVersion, fetchFn);
